_includes/nav.html

```diff
@@ -1,23 +1,21 @@
-<!-- <nav class="">
-  <div class="container">
-
-
-
-
-    <a href="/" {% if page.url == "/" %}class="active"{% endif %}>Home</a>
-    |
-    <a href="/about" {% if page.url == "/about" %}class="active"{% endif %}>About</a>
-
-  </div>
-</nav> -->
+<<nav id="nav"  class="navbar navbar-expand-sm ">
```

Comment:
Extra angle bracket results in this:

![image](https://user-images.githubusercontent.com/504505/53318127-183c7f80-3883-11e9-9b19-4d0ee139390c.png)

Reply:
You've noticed a lot of little things like this, which quite frankly come across as sloppy. I'm going to create a checklist before pushing to avoid things like this because they're a waste of your time, and also slow down the cadence.